temba/contacts/views.py

```
@@ -141,8 +145,10 @@ def clean_query(self):
                 raise forms.ValidationError(_('You cannot create a dynamic group based on "name" or "id".'))
         except forms.ValidationError as e:
             raise e
+        except SearchException as e:
+            raise forms.ValidationError(str(e))
 
-        except Exception as e:
+        except Exception as e:  # pragma: no cover
```

do we need this? if we can get here we should test it

---

we don't...

the question is do we want to blow up (and log Exception on Sentry) and possibly break UI/user session with a http_status=500, or show error to the user, which can then take a screenshot and create a problem report?

---

If we "don't" (don't is always theoretical!) then I think better not to catch anything here and on the off chance that we're wrong... then something ends up in sentry and we fix it. 

The problem with catching any random exception here and converting it to a Validation error is we might be swallowing a `AttributeError` and showing that to the user.

---

ok, then ... off with it's head ...